src/test/java/fr/inria/coming/spoon/repairability/repairtools/NPEfixTest.java

```diff
@@ -10,7 +10,7 @@
     @Test
     public void NPEfixTest() throws Exception {
         FinalResult result = TestUtills.runRepairability("NPEfix", "/repairability_test_files/NPEfix/");
-        TestUtills.numberOfInstances(result, 18, 16);
```

Comment:
I have to change this back to the previous state(master state) if I have to delete the test files.

Reply:
What if you add again the NpeFix test files?

Reply:
added.